### ISSUE TITLE
WebRTC audio input

### ISSUE THREAD

**Issue by Superlukec** [segmentio/nightmare#1189]
I want to create WebRTC peer A (nightmare). I have managed to establish the connection to other peer B (web browser). When I talk over peer A I can hear myself at peer B, but when I talk over peer B I cannot hear myself at peer A.

Do you have any propositions?

Thank you.

**Comment by JonathanBristow** [comment]
You might want to include sample code or a working demo.

**Comment by Superlukec** [comment]
Ok. This is my code for NightmareJS (server.js):

```
var Nightmare = require('nightmare');		
var nightmare = Nightmare({ 	
	show: false,     
	'webPreferences': {
		partition: 'persist:somesession'
	},
	switches: {	
		'ignore-certificate-errors': true
	}	
});

nightmare
  .goto('https://localhost:7000/')     // local website, which uses standard procedure to connect to the WebRTC
  .run(function(err, nightmare){
	  console.log('Website loaded.');
  });
  
// in case of CTRL+C
process.on("SIGINT", () => {
  console.log("Caught SIGINT. Exiting in 5 seconds.");
  
  nightmare
  .end()
  .then(function (result) {
      console.log("Closing the Nigtmare");
  }) 
  

  setTimeout(() => {
    console.log("Killing the process.");
    process.exit(0);
  }, 5000);
});
```

I'm testing this code on two environments (Windows and headless Linux Debian):

Windows
`node server.js`

Linux (here I am using xvfb)
`xvfb-run node server.js`

**Comment by casesandberg** [comment]
Are you still having trouble and have you tried 2.10.0? I am going to close this, but feel free to open a new issue if you are still having problems.

**Comment by Superlukec** [comment]
I didn't solved the problem, but I will try the 2.10.0 version and I will let you know if the problem still persist. Thank you very much.

**Comment by Superlukec** [comment]
@casesandberg I made a test with version 2.10.0. The problem still presists.
